dags/openshift_nightlies/util/manifest.py

```diff
@@ -15,23 +18,34 @@ def __init__(self, root_dag_dir):
         self.release_stream_base_url = var_loader.get_secret("release_stream_base_url")
         self.get_latest_releases()
 
-    def get_latest_releases(self): 
+    def get_latest_releases(self):
         release_streams = [ version['releaseStream'] for version in self.yaml['versions']]
+        print(release_streams)
```

Comment:
I think we can remove this print message, or, replace it by log.info

Reply:
Good catch. Will delete this one since it only prints what is in manifest.yaml. The releases that are resolved are printed elsewhere at: https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/dag.py#L296